src/components/checkboxes/checkboxes.js

```diff
@@ -40,7 +41,7 @@ Checkboxes.prototype.setAttributes = function ($input) {
   $input.setAttribute('aria-expanded', inputIsChecked)
 
   var $content = document.querySelector('#' + $input.getAttribute('aria-controls'))
-  $content.setAttribute('aria-hidden', !inputIsChecked)
+  $content.classList.toggle('govuk-checkboxes__conditional--hidden', !inputIsChecked)
```

Comment:
In the case that CSS fails but JS succeeds, should we also include 'aria-hidden'? Or is this a bit too over the top?

Reply:
I reckon this is fine as is.